### PR TITLE
Add 2.2.0.0 release notes

### PR DESCRIPTION
**Pull request by junqiu-lei**
Signed-off-by: Junqiu Lei <junqiu@amazon.com>

### Description
Add 2.2.0.0 release note 

### Issues Resolved
https://github.com/opensearch-project/k-NN/issues/467
 
### Check List
- [x] Commits are signed as per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/k-NN/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by martin-gaievski** [comment]
please add changes related to Lucene HNSW, we have merged them with rebase so it went as a list of individual commits/PRs https://github.com/opensearch-project/k-NN/pull/486/commits

**Comment by junqiu-lei** [comment]
Thanks @martin-gaievski, I added it with the [merged PR ](https://github.com/opensearch-project/k-NN/pull/484/files#diff-d2034efef09dc1f131d09aed4a112eabfad503cc2096daed4af1e4a439ea827fR8), or I can also instead to add the related PRs one by one.

File: release-notes/opensearch-knn.release-notes-2.2.0.0.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+## Version 2.2.0.0 Release Notes
+
+Compatible with OpenSearch 2.2.0
+
+### Enhancements
+* Add KNN codec that is based on Lucene92 codec([#444](https://github.com/opensearch-project/k-NN/pull/444))
+* Lucene Based k-NN search support([#486](https://github.com/opensearch-project/k-NN/pull/486/commits))
```
Comment (vamshin) [inline release-notes/opensearch-knn.release-notes-2.2.0.0.md:7]: Can we move this under ```Feature``` section?

Comment (junqiu-lei) [reply release-notes/opensearch-knn.release-notes-2.2.0.0.md]: sure, updated

File: release-notes/opensearch-knn.release-notes-2.2.0.0.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+## Version 2.2.0.0 Release Notes
+
+Compatible with OpenSearch 2.2.0
+
+### Features
+* Lucene Based k-NN search support([#486](https://github.com/opensearch-project/k-NN/pull/486/commits))
+
+### Enhancements
+* Add KNN codec that is based on Lucene92 codec([#444](https://github.com/opensearch-project/k-NN/pull/444))
+* Make inner product similarity not supported([#488](https://github.com/opensearch-project/k-NN/pull/488))
```
Comment (jmazanec15) [inline release-notes/opensearch-knn.release-notes-2.2.0.0.md:10]: Specify that this is for lucene only.

Comment (junqiu-lei) [reply release-notes/opensearch-knn.release-notes-2.2.0.0.md]: updated

**Review by jmazanec15 — APPROVED** [review]

File: release-notes/opensearch-knn.release-notes-2.2.0.0.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+## Version 2.2.0.0 Release Notes
+
+Compatible with OpenSearch 2.2.0
+
+### Features
+* Lucene Based k-NN search support([#486](https://github.com/opensearch-project/k-NN/pull/486/commits))
```
Comment (martin-gaievski) [inline release-notes/opensearch-knn.release-notes-2.2.0.0.md:6]: I main PR there are child PRs listed [485](https://github.com/opensearch-project/k-NN/pull/485), do we need link to commits?

**Review by martin-gaievski — APPROVED** [review]

**Review by vamshin — APPROVED** [review]
LGTM! Thanks

**Pull request merged**